### PR TITLE
set grub-installer/bootdev default only on jessie

### PR DESCRIPTION
**Pull request by mmoll**
closes: GH-154
@sathieu please test

### CODE REVIEW

File: preseed/provision.erb [hunk 1]
```diff
@@ -141,7 +141,8 @@ d-i grub-installer/only_debian boolean true
 d-i grub-installer/with_other_os boolean true
 <% if @host.params['install-disk'] -%>
 d-i grub-installer/bootdev string <%= @host.params['install-disk'] %>
-<% else -%>
+<% end -%>
+<% if @host.operatingsystem.name == 'Debian' and @host.operatingsystem.major.to_i == 8 -%>
```
Comment (sathieu) [inline preseed/provision.erb]: couldn't this be >= 8 ?

Comment (sathieu) [inline preseed/provision.erb]: this should probably be elsif?

Comment (sathieu) [inline preseed/provision.erb]: I would go for:
```
<% elif @host.operatingsystem.name == 'Debian' and @host.operatingsystem.major.to_i >= 8 -%>
```

Comment (mmoll) [inline preseed/provision.erb]: we can do that, but the thought was, that with Ubuntu/vivid the topic will bubble up again.

Comment (sathieu) [inline preseed/provision.erb]: It seems that Vivid has grub-installer (1.78ubuntu25) which doesn't support "default".

Refs:
https://code.launchpad.net/~ubuntu-branches/ubuntu/vivid/grub-installer/vivid
https://tracker.debian.org/media/packages/g/grub-installer/changelog-1.103

Comment (mmoll) [inline preseed/provision.erb]: We'll really this, when it's released. Maybe someone opens a bug against the Ubuntu installer and the Debian change gets imported (I hope ;)).

**Comment by mmoll** [comment]
updated with @sathieu's suggestions

**Comment by sathieu** [comment]
works for me

**Comment by mmoll** [comment]
@ekohl? @domcleal?

**Comment by domcleal** [comment]
Merged as 2a66cc2.  Thanks @mmoll and @sathieu!

**Pull request closed without merging**